LocalMultiplayerAgent/Controllers/SessionHostController.cs

```diff
@@ -78,22 +79,32 @@ public SessionHostController(IHostApplicationLifetime appLifetime)
                     config = Globals.SessionConfig;
                 }
 
+                sendMaintenanceEvent = (numHeartBeats == Globals.Settings.NumHeartBeatsForMaintenanceEventResponse);
+
                 HeartBeatsCount[sessionHostId]++;
             }
             else
             {
+                sendMaintenanceEvent = (numHeartBeats == 1);
```

Comment:
why are we doing this here?

Reply:
Changing this. Should've been checking NumHeartBeatsForMaintenanceEventResponse